invenio_communities/templates/semantic-ui/invenio_communities/base.html

```diff
@@ -8,3 +8,10 @@
 #}
 
 {% extends config.BASE_TEMPLATE %}
+
+{%- block css %}
+{{ super() }}
+{% if community %}
+<link rel="stylesheet" type="text/css" href="/communities/{{community.id}}/brand_theme.css">
```

Comment:
we should use here the community slug! Probably we can rename the `brand_theme.css` to `theme.css` or similar.